site.yml

```diff
@@ -99,13 +92,8 @@
   # todo: add a meta dependency on role sample-data
 
 - hosts: importers
-  tasks:
-    - name: restart importer service to make sure it works correctly with sample data
-      ansible.builtin.systemd:
-        name: "{{ product_name }}-importer"
-        state: restarted
-      become: yes
-      when: "not installation_mode == 'uninstall'"
+  roles:
```

Comment:
do we install the role importers twice? not necessary imho

Reply:
Only once, I moved the role execution